releasetool/data/roles/import_gpg_private/tasks/main.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 - name: import gpg key
-  shell: "gopass show '{{ pass_name_key }}' | gpg2 --homedir '{{ work_dir_gpg }}' --import"
+  shell: "gopass show '{{ pass_name_key }}' | gpg --homedir '{{ work_dir_gpg }}' --import"
```

Comment:
why was that needed?

Reply:
ah, on recent fedoras, g2 is a symlink to g, not gnupg1 anymore, nice

Reply:
I had some issues with gpg2 and importing, will switch it back